code/modules/clothing/under/shorts.dm

```diff
@@ -4,7 +4,7 @@
 	gender = PLURAL
 	body_parts_covered = GROIN
 	female_sprite_flags = NO_FEMALE_UNIFORM
-	mutantrace_variation = MUTANTRACE_VARIATION
+	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON
```

Comment:
Same here - all the shorts have a digitigrade icon already, they're the only unders that do in our code base - if this is intended, I may have misunderstood what `DIGITIGRADE_VARIATION_NO_NEW_ICON` means